tests/test_proc/test_whitening.py

```diff
@@ -0,0 +1,141 @@
+"""Tests for signal whitening."""
+import numpy as np
+import pytest
+
+from dascore import get_example_patch
+from dascore.exceptions import ParameterError
+from dascore.units import Hz
+
+
+class TestWhitening:
+    """Tests for the dispersion module."""
+
+    @pytest.fixture(scope="class")
+    def test_patch(self):
+        """Return a shot-record used for testing patch."""
+        test_patch = get_example_patch("dispersion_event")
+        return test_patch.resample(time=(200 * Hz))
+
+    def test_whitening(self, test_patch):
+        """Check consistency of test_dispersion module."""
+        # assert velocity dimension
+        whitened_patch = test_patch.whitening([10, 50], 5)
+        assert "distance" in whitened_patch.dims
+        # assert time dimension
+        assert "time" in whitened_patch.dims
+        assert np.array_equal(
+            test_patch.coords.get_array("time"), whitened_patch.coords.get_array("time")
+        )
+
+        assert np.array_equal(
+            test_patch.coords.get_array("distance"),
+            whitened_patch.coords.get_array("distance"),
+        )
+
+    def test_default_whitening(self, test_patch):
```

Comment:
Suggest changing the name to "test_default_whitening_no_input"

Reply:
Done